tree_clipper/id_data_getter.py

```diff
@@ -0,0 +1,61 @@
+import bpy
+
+from typing import Callable
+
+_ID_TYPE_TO_DATA_BLOCK: dict[str, bpy.types.bpy_prop_collection] = {
+    "ACTION": bpy.data.actions,
+    "ARMATURE": bpy.data.armatures,
+    "BRUSH": bpy.data.brushes,
+    "CACHEFILE": bpy.data.cache_files,
+    "CAMERA": bpy.data.cameras,
+    "COLLECTION": bpy.data.collections,
+    "CURVE": bpy.data.curves,
+    # "CURVES": ???,
```

Comment:
This is due the "old curve type" being an Object and the "new curve type" being "CURVES"